docs/user-guide/logs/pipeline-config.md

```diff
@@ -650,3 +658,37 @@ The result will be:
   "born_time": 2021-07-08 16:00:00
 }
 ```
+
+## Dispatcher
+
+The pipeline dispatcher routes requests to other pipelines based on configured
+field values. It is useful when multiple log types share a single source and
+need to be stored in separate tables with different structures.
+
+A sample configuration is like:
+
+```yaml
+dispatcher:
+  field: type
+  rules:
+    - value: http
+      table_suffix: http
+      pipeline: http
+    - value: db
+      table_suffix: db
+
+```
+
```

Comment:
please explain more about this example

Reply:
The content below is explaining how it will work.